nursery/debug-build.yml

```diff
@@ -7,4 +7,4 @@ rule:
   features:
     - or:
       - string: Assertion failed!
```

Comment:
shouldn't strings with white space be quoted as well?

Reply:
covered not that we enforce all strings start/end with double quotes.